pages/learn/manage/account.md

```diff
@@ -60,7 +60,7 @@ When an application needs to be shared with more than one user, the application
 
 ### Member types
 
-| Member Type   | Add members   | Delete App  | Add/Remove device  | Device actions  | Tags | Dev Env Variables | SSH access | Push | Fleet Env Variables | Fleet actions |
+| Member Type   | [Add members][administrator]   | Delete App  | Add/Remove device  | [Device actions][device-actions]  | [Tags][tags] | [Env Variables][env-variables] | [SSH access][ssh-access] | [Push][deploy] | [Fleet Variables][fleet-variables] | Fleet actions |
```

Comment:
From what I understand, a user with an Observer role can change Env Variables but not Fleet Variables. That said, the Env Variables vs Fleet Variables naming is a bit confusing based on where they're linked to. `[env-variables]:/learn/manage/serv-vars/` (https://www.balena.io/docs/learn/manage/serv-vars/) have a fleet sections, and `[fleet-variables]:/learn/manage/configuration/` (https://www.balena.io/docs/learn/manage/configuration/) do too. I'm not sure what the difference being conveyed is.

Reply:
@cywang117 I was confused on that one as well. 
Env variables is fine but the page it linked to is muddled up with a lot of variables. 
Fleet variables is the fleet configuration part but yes needs more accuracy, I will take a deeper look and try to separate them out.

Reply:
The observer is a read-only role, hence it being able to change env var only seems fishy. We should test this before making the change + with the change that is coming in regarding variables is bound to simplify this.

Reply:
> The observer is a read-only role

Hmm. Just looked at the [member roles table](https://www.balena.io/docs/learn/manage/account/#member-types) in the docs again and it says Observers can change "Dev Env Variables" -- sorry, I was unclear. Do you know what Dev Env Variables are @vipulgupta2048 ? I'm confused about the name.

Reply:
Dev Env Variables vs Fleet Env Variables makes me think Observers can change device-level env vars while Developers + Admins can change fleet-level, but yeah, that doesn't make sense based on Observer being a read-only role.

Reply:
Ah. Never mind, I just saw the proposed work in https://github.com/balena-io/docs/issues/1969. I see what you're talking about with reagrds to the incoming change.